pkg/cosign/upload.go

```diff
@@ -93,16 +92,30 @@ func UploadTLog(signature, payload []byte, pemBytes []byte) (string, error) {
 			}
 			fmt.Println("Signature already exists. Displaying proof")
 
-			return FindTlogEntry(rekorClient, cs.Base64Signature, cs.Payload, pemBytes)
+			uid, err := FindTlogEntry(rekorClient, cs.Base64Signature, cs.Payload, pemBytes)
```

Comment:
should the log entry proof be grabbed from here to get inclusion proofs for existing entries?